### PR TITLE
Feat/update price api

### PR DESCRIPTION
**Pull request by djhahe**

### CODE REVIEW

File: src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx [hunk 1]
```diff
@@ -1,20 +1,27 @@
 import { BoxProps } from '@chakra-ui/react';
 
 import TotalBalance from '@/components/Common/TotalBalance';
-import { useGetCoingeckoCoin } from '@/hooks/queries/useGetCoingeckoCoin';
+import { usePrice } from '@/hooks/queries/usePrice';
```
Comment (0xkynz) [inline src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx]: Should be useGetCSPRMarketData for more clear, with useGet as prefix for queries hook

File: src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
 
 export type AccountBalancesProps = BoxProps;
 
 const AccountTotalBalances = () => {
   const {
-    data: { marketCap, dailyVolume } = { marketCap: 0, dailyVolume: 0 },
-  } = useGetCoingeckoCoin();
+    data: { market_cap, volume_24h, circulating_supply, total_supply } = {
```
Comment (0xkynz) [inline src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx:10]: Should normalize:
1. circulating_supply -> circulatingSupply 
2. market_cap -> marketCap
...
snake_case to camelCase
in usePrice hook

File: src/modules/Home/components/AccountInfo/index.tsx [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+import { useSelector } from 'react-redux';
 
-import ButtonConnectWallet from '@/modules/core/ButtonConnectWallet';
+import { publicKeySelector } from '@/store/wallet';
 
 export type AccountInfoProps = BoxProps;
 
 const AccountInfo = (props: AccountInfoProps) => {
   const { t } = useTranslation();
+  const publicKey = useSelector(publicKeySelector);
```
Comment (0xkynz) [inline src/modules/Home/components/AccountInfo/index.tsx]: Should use useAccount hook instead of selector

File: src/utils/request.ts [hunk 4]
```diff
@@ -1,7 +1,11 @@
 import axios, { AxiosResponse } from 'axios';
+console.log(
+  '🚀 ~ file: request.ts:8 ~ import.meta.env.NEXT_PUBLIC_API_URL:',
+  import.meta.env
+);
 
 const request = axios.create({
-  baseURL: process.env.NEXT_PUBLIC_API_URL,
+  baseURL: import.meta.env.NEXT_PUBLIC_API_URL,
```
Comment (0xkynz) [inline src/utils/request.ts:4]: 👍 Ok for this sir.

File: src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx (hunk 2 above)
Comment (djhahe) [reply src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx:10]: I think we can follow API name fields, no need to map the name again which make code more complicated

**Review by 0xkynz — APPROVED** [review]

**Pull request merged**